src/actions.js

```diff
@@ -46,7 +46,7 @@ export default function actionsFactory(config) {
           }
           // Call AUTHENTICATE because it's utterly the same
           commit(types.AUTHENTICATE, constructUser(cognitoUser, session));
-          resolve();
+          resolve({currentUser: cognitoUser});
```

Comment:
Thanks for pointing this out but it's better to return a user object built with constructUser method instead of a native Cognito user object